ref_arch/multi_tenant.rst

```diff
@@ -0,0 +1,590 @@
+    company_id bigint,   -- added
+    ad_id bigint,        -- was: REFERENCES ads (id),
+    clicked_at timestamp without time zone NOT NULL,
+    site_url text NOT NULL,
+    cost_per_click_usd numeric(20,10),
+    user_ip inet NOT NULL,
+    user_data jsonb NOT NULL,
+    PRIMARY KEY (company_id, id),      -- added
+    FOREIGN KEY (company_id, ad_id)    -- added
+      REFERENCES ads (company_id, id)
+  );
+
+  CREATE TABLE impressions (
+    id bigserial,         -- was: PRIMARY KEY
+    company_id bigint,    -- added
+    ad_id bigint,         -- was: REFERENCES ads (id),
+    seen_at timestamp without time zone NOT NULL,
+    site_url text NOT NULL,
+    cost_per_impression_usd numeric(20,10),
+    user_ip inet NOT NULL,
+    user_data jsonb NOT NULL,
+    PRIMARY KEY (company_id, id),       -- added
+    FOREIGN KEY (company_id, ad_id)     -- added
+      REFERENCES ads (company_id, id)
+  );
+
+Distributing Tables, Ingesting Data
+-----------------------------------
+
+.. note::
+
+  This guide is designed so you can follow along in your own Citus database. Use one of these alternatives to spin up a database:
+
+  * Run Citus locally using :ref:`single_machine_docker`, or
+  * Provision a cluster using `Citus Cloud <https://console.citusdata.com/users/sign_up>`_
+
+  You'll run the SQL commands using psql:
+
+  * **Docker**: :code:`docker exec -it citus_master psql -U postgres`
+  * **Cloud**: :code:`psql "connection-string"` where the connection string for your formation is available in the Cloud Console.
+
+  In either case psql will be connected to the coordinator node for the cluster.
+
+At this point feel free to follow along in your own Citus cluster by `downloading <https://examples.citusdata.com/tutorial/schema.sql>`_ and executing the SQL to create the schema. Once the schema is ready, we can tell Citus to create shards on the workers. From the coordinator node, run:
+
+::
+
+  SELECT create_distributed_table('companies',   'id');
+  SELECT create_distributed_table('campaigns',   'company_id');
+  SELECT create_distributed_table('ads',         'company_id');
+  SELECT create_distributed_table('clicks',      'company_id');
+  SELECT create_distributed_table('impressions', 'company_id');
+
+The :ref:`create_distributed_table` function informs Citus that a table should be distributed and that future incoming queries to those tables should be planned for distributed execution. The function also creates shards for the table on worker nodes, allowing incoming data and queries to be routed there.
+
+The next step is loading sample data into the cluster.
+
+.. code-block:: bash
+
+  # download and ingest datasets from the shell
+
+  for dataset in companies campaigns ads clicks impressions; do
+    curl -O https://examples.citusdata.com/mt_ref_arch/${dataset}.csv
+  done
+
+.. note::
+
+  **If you are using Docker,** you should use the :code:`docker cp` command to copy the files into the Docker container.
```

Comment:
Good point. On my own machine I ran the native psql and `\copy` moved the data inside to postgres, but the docker instructions that we provide have the user run psql inside the container (since it may not exist on the regular user machine).